orphanarium-aeson/src/Orphanarium/Aeson/Time.hs

```diff
@@ -0,0 +1,15 @@
+module Orphanarium.Aeson.Time (
+    genTime
+  ) where
+
+import           Control.Applicative
+
+import           Data.Time
+
+import           Test.QuickCheck
+
+
+genTime :: Gen UTCTime
+genTime = UTCTime
+  <$> fmap ModifiedJulianDay arbitrary
+  <*> fmap secondsToDiffTime (choose (0, 60 * 60 * 24))
```

Comment:
> What about millis?

`fmap secondsToDiffTime (choose (0, 60 * 60 * 24 * 1000))`

    Left "could not parse ISO-8601 date"